src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -340,10 +340,14 @@ private void registerAutoCommands(){
     NamedCommands.registerCommand("Eject", new Eject(intakeShooter));
 
     //  NamedCommands.registerCommand("ArmToSpeaker", new MoveToPos(arm, Armc.SPEAKER_ANGLE_RAD, 0));
-    NamedCommands.registerCommand("ArmToAmp", new ArmToPos(arm, Armc.AMP_ANGLE_RAD,0));
-    NamedCommands.registerCommand("ArmToSubwoofer", new ArmToPos(arm, Armc.SUBWOOFER_ANGLE_RAD,1));
-    NamedCommands.registerCommand("ArmToPodium", new ArmToPos(arm, Armc.PODIUM_ANGLE_RAD, 2));
-    NamedCommands.registerCommand("ArmToGround", new ArmToPos(arm, GROUND_INTAKE_POS, 1));
+    NamedCommands.registerCommand("ArmToAmp",
+        new ArmToPos(arm, Armc.AMP_ANGLE_RAD, AMP_RPM_INDEX));
```

Comment:
Can you combine the position + index? Ideally you can just do `new ArmToPos(arm, pos)` and it should take care of the rest instead of having to add additional parameters.